src/app/router.ts

```diff
@@ -72,7 +72,7 @@ const router = new Router([
   {
     action: statement.viewStatement,
     name: 'admin.statement.view',
-    path: '/organisations/:organizationGUID/statements',
+    path: '/organisations/:organizationGUID/statements/:rangeStart',
```

Comment:
It took me a while to figure out the behaviour, where:

- `2018-05-01` gave today only with costs
- `2018-05-02` gave tomorrow only with no costs
- `2018-04-01` gave last month in full
- `2018-04-15` gave a partial month

Could this take a month rather than a start date and always report on the calendar month?

Reply:
The new error message doesn't make it to the browser, but hopefully that'll be fixed in https://www.pivotaltracker.com/story/show/156760174